# z-Test

In [6]:
from scipy.stats import norm
import numpy as np
from scipy.stats import t

In [5]:
1-norm.cdf(x=168, loc=164, scale=8/np.sqrt(150))

4.5706494145036913e-10

Den Mittelwert $ x 150 = 168cm $ oder einen noch extremeren Mittelwert zu beobachten, ist unter der Annahme der Nullhypothese $ µ0 = 164cm $ extrem unwahrscheinlich. Wir verwerfen somit die Nullhypothese.

# t-Test

In [8]:
1-t.cdf(x=168, df=149, loc=164, scale=10/np.sqrt(150))

1.241988350275669e-06

Der P-Wert wäre also 2 · 1.241988 · 10 −6 . Auch in diesem Fall können wir die Nullhypothese auf dem Signifikanzniveau $ α = 0.05 $ verwerfen.